test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/IntegrationTests/CodeGenerationIntegrationTest.cs

```diff
@@ -28,10 +28,7 @@ public class CodeGenerationIntegrationTest : IntegrationTestBase
         public void InvalidNamespaceAtEOF_Runtime()
         {
             var references = CreateCompilationReferences(CurrentMvcShim);
-            RunRuntimeTest(references, expectedErrors: new[]
-            {
-                "Identifier expected"
```

Comment:
Since we don't call into user code if a directive is invalid we no longer generate an empty namespace declaration (hence no error).